src/components/TodoItem.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
not critical, but you may add `aria-label` attribute in the place that causes the lint error
generally - try to figure out with lint error before disabling it - maybe you can fix it w/o disabling

Reply:
it`s from origin branche, i just cope it 
and comment befor too